recipe/meta.yaml

```diff
@@ -16,23 +16,22 @@ source:
 build:
   number: 0
   script: python -m pip install --no-deps --ignore-installed .
+  noarch: python
 
 requirements:
   build:
-    - python
+    - python >=3.5
     - pip
     - pbr !=2.1.0,>=2.0.0
 
   run:
-    - python
+    - python >=3.5
     - pbr !=2.1.0,>=2.0.0
-    - cmd2 !=0.8.3,<0.9.0  # [py2k]
-    - cmd2 !=0.8.3  # [py3k]
-    - prettytable <0.8,>=0.7.2
+    - cmd2 !=0.8.3
+    - prettytable >=0.7.2
```

Comment:
```suggestion
    - prettytable >=0.7.2,<0.8
```

Reply:
Same remark as in https://github.com/conda-forge/staged-recipes/pull/11675#discussion_r427752664

I am not convinced that these upper constraints are justified.

Reply:
Just following the metadata provided by the `cliff`` package

Reply:
May I suggest that for now we remove the upper constraints: if there is a problem running these packages against the latest version of the dependencies, I will gladly reinstate the constraints.